Karma/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_ENVIRONMENT": "Production",
```

Comment:
Will it disable debugging features?

Reply:
Yes it will not provide exception information (will show different view) so its better to set it back to Development when we are not presenting our product.